loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -1189,7 +1189,7 @@ export class GLTFLoader implements IGLTFLoader {
         this.logOpen(`${context} ${camera.name || ""}`);
 
         this._babylonScene._blockEntityCollection = this._forAssetContainer;
-        const babylonCamera = new FreeCamera(camera.name || `camera${camera.index}`, Vector3.Zero(), this._babylonScene, false);
+        const babylonCamera = new TargetCamera(camera.name || `camera${camera.index}`, Vector3.Zero(), this._babylonScene, false);
```

Comment:
why targetcamera? It is supposed to be the parent of free and arcrotate

Reply:
Plus it is a breaking change

Reply:
I'm going to revert this for now. Creating a free camera this way doesn't work right because the transform of the camera is on a parent transform node instead of the camera and thus the panning controls for the free camera is wrong if the camera transform has rotation in the glTF. This change was trying to prevent the camera from moving.